step-controller/step-controller-server/src/main/java/step/core/Controller.java

```diff
@@ -68,7 +68,7 @@
 
 public class Controller {
 
-	public static final Version VERSION = new Version(3,24,0);
+	public static final Version VERSION = new Version(3,24,3);
```

Comment:
So this is how you smuggle in a version bump... :)

Reply:
indeed :-)